docs/source/configuration/index.rst

```diff
@@ -14,3 +14,103 @@ The AtlasDB configuration has two main blocks - keyValueService and leader. Plea
    cassandra_KVS_configuration
    leader_config
    logging
+
+There are three valid configuration modes, listed below.
+
+Leader block
+------------
+
+This configuration setup is required to use Cassandra KVS with more than one node.
+
+A minimal AtlasDB configuration for running against cassandra will look like the below.
+
+Importantly - your lock creator must be consistent across all nodes.
```

Comment:
Nit: could use a `.. caution::` (or warning?) [box](http://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#boxes) to make this stand out.